fault-proof/src/proposer.rs

```diff
@@ -133,7 +133,7 @@ where
     }
 
     #[tracing::instrument(name = "[[Proving]]", skip(self), fields(game_address = ?game_address))]
-    pub async fn prove_game(&self, game_address: Address) -> Result<TxHash> {
+    pub async fn prove_game(&self, game_address: Address) -> Result<(TxHash, u64, u64)> {
```

Comment:
add a doc comment here about what the u64s mean? in my experience outputting tuples gets confusing fast